Assets/MRTK/Providers/Oculus/XRSDK/OculusXRSDKDeviceManagerProfile.cs

```diff
@@ -117,33 +117,18 @@ public OVRHand.TrackingConfidence MinimumHandConfidence
         /// Current tracking confidence of right hand. Value managed by OculusQuestHand.cs.
         /// </summary>
         public OVRHand.TrackingConfidence CurrentRightHandTrackingConfidence { get; set; }
-#endif
-
-        [SerializeField]
-        [Range(0f, 5f)]
-        [Tooltip("Time after which low confidence is considered unreliable, and tracking is set to false. Setting this to 0 means low-confidence is always acceptable.")]
-        private float lowConfidenceTimeThreshold = 0.2f;
-
-        /// <summary>
-        /// Time in seconds after which low confidence is considered unreliable, and tracking is set to false.
-        /// </summary>
-        public float LowConfidenceTimeThreshold
-        {
-            get => lowConfidenceTimeThreshold;
-            set => lowConfidenceTimeThreshold = value;
-        }
 
         [Header("Performance Configuration")]
         [SerializeField]
         [Tooltip("Default CPU performance level (0-2 is documented), (3-5 is undocumented).")]
         [Range(0, 5)]
-        private int defaultCpuLevel = 2;
+        private OVRManager.ProcessorPerformanceLevel defaultCpuLevel = OVRManager.ProcessorPerformanceLevel.SustainedHigh;
 
         /// <summary>
         /// Accessor for the Oculus CPU performance level.
         /// https://developer.oculus.com/documentation/native/android/mobile-power-overview
         /// </summary>
-        public int CPULevel
+        public OVRManager.ProcessorPerformanceLevel CPULevel
```

Comment:
Also, were these changes intended for this PR? I don't see a reference to them in the description

Reply:
Snuck these into this PR by accident, these are for compatibility with the latest Oculus package.